setup.py

```diff
@@ -35,31 +34,34 @@
 
 PY3 = sys.version_info[0] == 3
 
-with io.open('README.md', encoding='utf-8') as f:
-    long_description = f.read()
+try:
+    FileNotFoundError
+except NameError:
+    FileNotFoundError = IOError
 
+try:
+    with io.open('README.md', encoding='utf-8') as f:
+        long_description = f.read()
+except FileNotFoundError:
+    long_description = ''
 
-class Tox(TestCommand):
-    user_options = [('tox-args=', None, "Arguments to pass to tox")]
+
+class CleanCommand(Command):
+    """Custom clean command to tidy up the project root."""
+    user_options = []
 
     def initialize_options(self):
-        TestCommand.initialize_options(self)
-        self.tox_args = ''
+        pass
 
     def finalize_options(self):
-        TestCommand.finalize_options(self)
-        self.test_args = []
-        self.test_suite = True
+        pass
 
-    def run_tests(self):
-        # import here, cause outside the eggs aren't loaded
-        import tox
-        errno = tox.cmdline(args=self.tox_args.split())
-        sys.exit(errno)
+    def run(self):
+        os.system('rm -vrf ./build ./dist ./*.pyc ./*.tgz ./*.egg-info')
```

Comment:
Not sure that `*.tgz` should be in here - that isn't a file that the build/setup.py produces is it? (as those would live under build/ or dist/)

Reply:
To be cleaned-up indeed.